### ISSUE TITLE
[Bug Report]: `frozenId` prop of MoveChoiceOverlay should default to null

### ISSUE THREAD

**Issue by itsalaidbacklife** [cuttle-cards/cuttle#94]
### Bug Summary

Currently the `frozenId` prop is required but instead it should default to null

### Detailed Description

This results in the following console error:
![](https://media.discordapp.net/attachments/833700472774852628/1020486420403601449/unknown.png)

We should update the `MoveChoiveOverlay` so the `frozenId` prop is not required and has a default value of `null`

### Steps to Reproduce

1.
2.
3.

**Comment by Jayaram18** [comment]
Hi @itsalaidbacklife ,
      I would like to work on this issue. I believe that we need to remove required field and add default instead. 
Please provide further details on how to test it before raising a PR.

**Comment by itsalaidbacklife** [comment]
> Hi @itsalaidbacklife ,
> 
>       I would like to work on this issue. I believe that we need to remove required field and add default instead. 
> 
> Please provide further details on how to test it before raising a PR.

Great! So the issue arises while inside a game. It will trigger an error in the console whenever you click a card in your hand to view the available move choices. 

So first you'll want to verify the issue on your machine by booting the server and client locally, then opening two different browsers and navigating each of them to localhost:8080, signing up with two accounts, creating and joining a game, hitting the ready button, and then once the game loads, clicking a card in your hand and confirming that the above error is logged to the console.

Then you can implement the fix as described above and rerun the above steps to confirm that the error is no longer output to the console. Please feel free to ask any other questions as they arise.

**Comment by itsalaidbacklife** [comment]
@Jayaram18 are you still interested in fixing this issue?

**Comment by Jayaram18** [comment]
I'm currently unavailable. Sorry

**Comment by byarmis** [comment]
Looking at [Vue's documentation](https://vuejs.org/guide/components/props.html#prop-validation), it says:

> All props are optional by default, unless `required: true` is specified.

In `MoveChoiceOverlay.vue`, `frozenId` is defined as follows:

```
...
frozenId: {
    type: Number,
    required: true,
},
...
```
By my count, `frozenId.*null` shows up in 11 different places and makes logical sense vs setting `frozenId` to be zero.

----

I propose removing the `required: true` line from `frozenId`'s definition in `MoveChoiceOverlay`

**Comment by byarmis** [comment]
This can be closed

**Comment by itsalaidbacklife** [comment]
Closed by #150
